Resources/lang/translation/fr/translations.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+return [
+    'title' => [
+        'translations' => 'Traductions',
+        'edit translation' => '&Eacute;diter une traduction',
+    ],
+    'button' => [
+        'create translation' => 'Cr&eacute;er une traduction',
+    ],
+    'table' => [
+    ],
+    'form' => [
+    ],
+    'messages' => [
+    ],
+    'validation' => [
+    ],
+    'Clear translation cache' => 'Nettoyer le cache des traductions',
+    'Export' => 'Exporter',
+    'Import' => 'Importer',
+    'Import csv translations file' => 'Importer un fichier de traductions',
+    'only for developers' => 'N&eacute;cessitera un gestion git !',
```

Comment:
You can use the usual UTF8 characters. Meaning é, è etc. are all ok. Use a single quote by escaping it `\'`